src/main/java/DCInvestigatorPlugin.java

```diff
@@ -185,14 +173,29 @@ private ErrorReport checkValidityofOutputVariable(OutputVariableImpl vv, PairedD
             //numCurves = numLifecyclesPerReal;
         }
         int prevLifecycles = real*numLifecyclesPerReal;
+
+        //Extracts realization number from dss record
+        String initialSim= outPdc.toString();
+        String[] simParse = initialSim.split("/",0);
+        String currentRealization = simParse[6].substring(2,8);
+        Integer curRealization = Integer.parseInt(currentRealization);
+
+        //Stores realization number in arraylist
+        ArrayList<Integer> realization = new ArrayList<Integer>();
+        realization.add(curRealization);
```

Comment:
I'm not sure I follow why we're storing that integer in a list rather than just as an Int. What's the thought there?

Reply:
it was my hacky way of trying to get the first realization number that is run by the DC.

Reply:
I just mean that right above it we store it as an integer(LN181), then we put that integer in an arrayList(LN185) then we pull that single integer back in (LN188) to multiply it. Why not skip that arrayList step and just use the integer.